glusterd2/commands/volumes/commands.go

```diff
@@ -108,6 +108,13 @@ func (c *Command) Routes() route.Routes {
 			Version:      1,
 			ResponseType: utils.GetTypeString((*api.VolumeStatusResp)(nil)),
 			HandlerFunc:  volumeStatusHandler},
+		route.Route{
+			Name:         "VolumeStatusAll",
+			Method:       "GET",
+			Pattern:      "/volumes/{volname}/status-all",
```

Comment:
volname is not required, Instead you could use just `/volumes` or 
` /volumes/status`

Reply:
`/volumes/status` would be better I believe.